docs/team/mep.md

```diff
@@ -0,0 +1,265 @@
+MMTk Enhancement Proposal (MEP)
+===============================
+
+An MMTk Enhancement Proposal (MEP) is a more formal variant of issue.  It has a special format, and
```

Comment:
I used your sentence as the definitive description.  But I think we should still include the details here because this paragraph is the summary of this document.  It helps the reader (especially those who have just heard of MEP) understand important details without going through the whole document.